### ISSUE TITLE
Dict and array variables are not correctly highlited when accessing it

### ISSUE THREAD

**Issue by himperion92** [nokia/RED#194]
I'm using Eclipse Oxygen v2.0 and RED v0.8.3. When trying to access to a dict or array item directly, the highlight is not correct. Example:

```
@{response}=	Create List	${0x00}	${0x01}
	Should Be Equal	${response}[0]	${0x00}
```
As it can be seen, the index is not highlited as part of the variable:
![image](https://user-images.githubusercontent.com/33331691/37760052-d8435bb6-2db5-11e8-863a-66334546012b.png)

**Comment by adam-wypych** [comment]
this was never supported as far as I remember and I even do not expect that RIDE or any other editor support such behaviour. The index is not the part of the variable name.

**Comment by KrzysztofJozefowicz** [comment]
Thanks, it is still visible in 0.8.4, we have a task about fixing colouring #187 which is ongoing, I hope this could be fixed together.

**Comment by adam-wypych** [comment]
Just be aware that there is plan to change the way how variables can be accessed in RF 3.0.3 - the issue with further information https://github.com/robotframework/robotframework/issues/2600 .

**Comment by adam-wypych** [comment]
The colouring of the variable works as expected, when user follow the convention of incorporate index of item inside curly brackets like i.e. `${response[0]}`.

**Comment by michalanglart** [comment]
fixed in 0.8.5
